### PR TITLE
[DEV-10913] Update the disaster agency endpoints to use the covid_faba_spending table

### PR DESCRIPTION
**Pull request by aguest-kc**
**Description:**
Update the agency disaster endpoints to use new `covid_faba_spending` database table instead of the Elasticsearch index.

- `/api/v2/disaster/agency/loans/` 

- `/api/v2/disaster/agency/spending/`

**Technical details:**
Update the agency disaster endpoints to use new `covid_faba_spending` database table instead of the Elasticsearch index.

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
3. [x] Necessary PR reviewers:
    - [x] Backend
6. [x] Data validation completed
8. [x] Jira Ticket [DEV-10913](https://federal-spending-transparency.atlassian.net/browse/DEV-10913):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected (API | Script | Download)
    - [x] Before / After data comparison

**Area for explaining above N/A when needed:**
```
2. API documentation updated
The API contracts are not changing due to this change.

4. Matview impact assessment completed
No matviews are affected by this change.

5. Frontend impact assessment completed
The frontend is not impacted by this change.

7. Appropriate Operations ticket(s) created
No operations tickets are needed as part of this change.
```

### CODE REVIEW

**Comment by aguest-kc** [comment]
🛑 Do not merge this unless PR #4125 has been merged, because this uses the new model created in that PR 🛑

File: usaspending_api/disaster/v2/views/agency/loans.py
Comment (dporth-frb) [inline usaspending_api/disaster/v2/views/agency/loans.py:86]: See your other PR about my comments regarding implicitness.

File: usaspending_api/disaster/v2/views/agency/loans.py
Comment (dporth-frb) [inline usaspending_api/disaster/v2/views/agency/loans.py:108]: Good inline comment. It does not tell the viewer what the code does, but tells us why it's an empty list.

File: usaspending_api/disaster/v2/views/agency/spending.py
Comment (dporth-frb) [inline usaspending_api/disaster/v2/views/agency/spending.py:125]: See your other PR about my comments regarding implicitness.

File: usaspending_api/disaster/v2/views/agency/spending.py
Comment (dporth-frb) [inline usaspending_api/disaster/v2/views/agency/spending.py:128]: See your other PR about my comments regarding implicitness.

**Review by dporth-frb — CHANGES_REQUESTED** [review]

File: usaspending_api/disaster/v2/views/agency/loans.py
Comment (aguest-kc) [reply usaspending_api/disaster/v2/views/agency/loans.py]: Updated the `None` checks to be explicit.

**Review by dporth-frb — APPROVED** [review]

**Pull request merged**
